### DISCUSSION TITLE
Two short questions

### DISCUSSION THREAD

**Discussion by GameboyColor32** [jakobhellermann/bevy-inspector-egui#138] [Q&A] [answered]
Is there a way to change the entity names somehow?
Is there a way to drag and drop entities from the hierarchy? Let's say for changing the order of the hierarchy.

Thanks a lot for this awesome plugin !

**Comment by jakobhellermann** [comment, marked as answer]
1. Yes, you can just insert the `Name` component and it should be displayed that way
2. Nope, that currently isn't possible

**Reply by GameboyColor32** [reply]
thanks !
